test-stack/prepare-test-env.js

```diff
@@ -0,0 +1,46 @@
+import { writeFileSync, mkdirSync } from "fs";
+import { join } from "path";
+
+const forkUrl = process.env.BELLECOUR_FORK_URL || "https://bellecour.iex.ec";
+
+fetch(forkUrl, {
+  method: "POST",
+  body: JSON.stringify({
+    jsonrpc: 2.0,
+    method: "eth_blockNumber",
+    params: [],
+    id: 1,
+  }),
+})
+  .then((res) => res.json())
+  .then((jsonRes) => {
+    const forkBlockNumber = parseInt(jsonRes.result.substring(2), 16);
+    if (process.env.DRONE) {
+      const LOCAL_STACK_ENV_DIR = "local-stack-env";
+      console.log(
+        `Creating ${LOCAL_STACK_ENV_DIR} directory for drone test-stack`
+      );
+      mkdirSync(LOCAL_STACK_ENV_DIR, { recursive: true });
```

Comment:
Use https://github.com/google/zx?tab=readme-ov-file#-zx?

Reply:
It is useless to add a dependency on `zx` to do something nodejs can do natively